content/en/docs/openshift-portability/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Differences between Microshift and OKD"
+weight: 100
+main_menu: true
+content_type: concept
+description: "Addressing some innate differences between OKD and Microshift."
+---
+# Differences Between OKD and Microshift
+
+## App Portability
```

Comment:
Starting from the angle of "trimming down OKD" comes across as "OKD is too bloated" and neglects that OpenShift engineering is already investing a lot in footprint reduction.

Instead, can we explain that MicroShift's design goals + choices differs from OpenShift's in a few key aspects, namely:
* It's an application you can install on Linux, not a full-stack, self-managing solution like OpenShift. Users need to take care of managing the OS and its configuration themselves.
* It's meant as _minimal_ runtime for OpenShift/K8s workloads, not as full-featured build/dev environment. If users want more, they need to install it themselves.

The _consequence_ is that we while we strive for portability of edge workloads, there are limitations that we want to make the users aware of.